API.md

```diff
@@ -10,15 +10,21 @@ const Log = require('ipfs-log')
 
 ### Constructor
 
-#### new Log(ipfs, [id])
+#### new Log(ipfs, access, identity, [id])
 
 Create a log. Each log gets a unique ID passed as an argument. Returns a `Log` instance.
 
 ```javascript
-const log = new Log(ipfs, 'logid')
+const { AccessController, IdentityProvider } = Log
+const keystore = require('orbit-db-keystore').create('./keystorepath')
+const identity = await IdentityProvider.createIdentity(keystore, 'peerid')
+const ipfs = new IPFS()
+const access = new AccessController()
```

Comment:
This is really a nitpick comment and a tiny thing, but here we could swap the "ipfs" and "access" line to group the related variable together more closely (keystore, identity, access). Feel free to ignore this and not do that if you want :)

Reply:
This has been resolved 👍